VERSION

```diff
@@ -1 +1 @@
-v0.8.0
+v0.8.1
```

Comment:
github is reporting something strange with the formatting. Maybe you removed the newline character at the end of the file?